src/foam/comics/DAOControllerView.js

```diff
@@ -66,18 +83,29 @@ foam.CLASS({
       class: 'FObjectProperty',
       of: 'foam.comics.DAOController',
       name: 'data',
-      expression: function(importedData) { return importedData; }
+      expression: function(importedData) {
+        return importedData;
+      }
     },
     {
       name: 'cls',
-      expression: function(data) { return data.cls_; }
+      expression: function(data) {
+        return data.cls_;
+      }
     },
     {
       name: 'summaryView',
       factory: function() {
-        return this.importedSummaryView$ ?
-            this.importedSummaryView :
-            { class: 'foam.u2.view.ScrollTableView' };
+        var defaultView = { class: 'foam.u2.view.ScrollTableView' };
+        if (
+          Array.isArray(this.data.contextMenuActions) &&
+          this.data.contextMenuActions.length > 0
+        ) {
+          defaultView.contextMenuActions = this.data.contextMenuActions;
+        }
+        return this.importedSummaryView ?
```

Comment:
easier:
return this.importedSummaryView || defaultView;

Reply:
Resolved by 5a50e09.